tests/math/unary/test_nondifferentiable.py

```diff
@@ -51,3 +51,23 @@ def test_argmax(a, data):
 
     # array input
     assert_allclose(mg.argmax(a, axis=axis), np.argmax(a, axis=axis))
+
+
+@given(
+    a=hnp.arrays(
+        shape=hnp.array_shapes(max_side=4, max_dims=5), dtype=dtype_strat_numpy
+    ),
+    data=st.data(),
+)
+def test_any(a, data):
+    axis = data.draw(valid_axes(ndim=a.ndim, single_axis_only=True), label="axis")
```

Comment:
Why do you have `single_axis_only=True` here? We want to also test multiple axes

Reply:
Honestly, I have almost no idea how to use Hypothesis...I just consulted a couple tutorials and mostly copy-pasted the tests from above. I'm working on learning it so I can write better tests in the future - thank you for pointing this out!